File: snippets/contact-center/create-worker/create-worker.py
from twilio.rest import Client
import json

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
client = Client(account_sid, auth_token)

worker = client.taskrouter \
               .workspaces('WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX') \
               .workers \
               .create(attributes=json.dumps({
                    skills: 'support'
                }), friendly_name='Support Worker 1')

print(worker.sid)

File: snippets/contact-center/create-worker/create-worker.js
const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
const authToken = 'your_auth_token';
const client = require('twilio')(accountSid, authToken);

client.taskrouter.workspaces('WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                 .workers
                 .create({attributes: JSON.stringify({
                      'skills': 'support'
                  }), friendlyName: 'Support Worker 1'})
                 .then(worker => console.log(worker.sid))
                 .done();
